src/rdd/co_grouped_rdd.rs

```diff
@@ -181,19 +185,19 @@ impl<K: Data + Eq + Hash> RddBase for CoGroupedRdd<K> {
         //        )
     }
 }
-impl<K: Data + Eq + Hash> Rdd<(K, Vec<Vec<Box<dyn AnyData>>>)> for CoGroupedRdd<K> {
+
+use crate::rdd::rdd_rt::CoGroupedOutput;
+
+impl<K: Data + Eq + Hash> Rdd<(K, CoGroupedOutput)> for CoGroupedRdd<K> {
```

Comment:
For PairRdd, type alias seems good and reasonable, but I think it is better to keep this explicit in this definition. This basically merges the value collection of two different RDDs having same key type. Vec<Vec<Box<AnyData>>>can make things clear when someone looks at the definition.